src/commands/moderation/prefixCommands/functions/deleteCommand.ts

```diff
@@ -0,0 +1,105 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommand, Logger, makeEmbed, clearSinglePrefixCommandCache } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Delete Command - No Connection',
+    description: 'Could not connect to the database. Unable to delete the prefix command.',
+    color: Colors.Red,
+});
+
+const failedEmbed = (commandId: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Command - Failed',
+    description: `Failed to delete the prefix command with id ${commandId}.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (command: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Command - Does not exist',
+    description: `The prefix command ${command} does not exists. Can not delete it.`,
```

Comment:
```suggestion
    description: `The prefix command ${command} does not exists. Cannot delete it.`,
```
"Cannot" is generally more common. I suggest we use that but obviously that's personal preference and I'm happy to leave it as is!